program/c/src/oracle/upd_aggregate.h

```diff
@@ -134,14 +134,6 @@ static inline void upd_twap(
 // update aggregate price
 static inline bool upd_aggregate( pc_price_t *ptr, uint64_t slot, int64_t timestamp )
 {
-  // only re-compute aggregate in next slot
-  if ( slot <= ptr->agg_.pub_slot_ ) {
-    return false;
-  }
```

Comment:
this doesn't happen because of `if clock.slot > latest_aggregate_price.pub_slot_` check in rust `upd_price.rs`

Reply:
Seems good to remove this since it will need to go away for same slot agg